distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Type_Mapping.enso

```diff
@@ -64,7 +60,12 @@ type SQLite_Type_Mapping
             Value_Type.Date -> unsupported_date_time
             Value_Type.Date_Time _ -> unsupported_date_time
             Value_Type.Binary _ _ -> SQLite_Types.blob
-            Value_Type.Mixed -> SQLite_Types.text
+            Value_Type.Mixed ->
+                ## The best we could do would be to store mixed values and
+                   report invalid type or coerce the values to Text. Both seem
+                   bad when Mixed type is explicitly requested. Instead, we
+                   should rely on explicit casting to text.
+                Error.throw (Unsupported_Database_Operation.Error "SQLite tables do not support Mixed types.")
```

Comment:
`Mixed` columns, which are actually a single type should be supported (but that's another ticket and based on some stuff I have).

Reply:
Could you clarify? What is the meaning of a single-type column being mixed in the Database?

More importantly, we have no way of expressing this using SQL concepts, as far as I can tell.

Reply:
Fair - I was meaning that if I try to insert from an in-memory table with mixed values which arent mixed I can materialise the type. But guess best to be explicit to do this.